rllib/policy/eager_tf_policy_v2.py

```diff
@@ -667,8 +670,26 @@ def get_initial_state(self):
         return []
 
     @override(Policy)
-    def get_state(self):
+    @OverrideToImplementCustomLogic_CallToSuperRecommended
+    def get_state(self) -> PolicyState:
+        # Legacy Policy state (w/o keras model and w/o PolicySpec).
         state = super().get_state()
+
+        # Add this Policy's spec so it can be retreived w/o access to the original
```

Comment:
Add PolicySpec AND - if possible - native keras/torch saved model files to returned state.